spec/controllers/home_controller_spec.rb

```diff
@@ -328,7 +338,15 @@ def wrap_process_args(*args)
         frames = Rollbar.last_report[:body][:trace][:frames]
 
         expect(frames[-1][:locals]).to be_eql(locals[0])
-        expect(frames[-2][:locals]).to be_eql(locals[1])
+
+        if Gem::Version.new(RUBY_VERSION) < Gem::Version.new('3.0.0')
+          expect(frames[-2][:method]).to be_eql('tap')
+          expect(frames[-2][:locals]).to be_eql(locals[1])
+        else
+          expect(frames[-2][:method]).to be_eql('tap')
+          expect(frames[-2][:locals]).to be_nil
+        end
```

Comment:
> is there any particular reason to keep the redundant except?

Since the frames have differences in different Rubies, the `expect(frames[-2][:method]).to be_eql('tap')` verifies we know that it is the same frame, not that an additional frame is present or some other difference.